locale/tr.po

```diff
@@ -17989,19 +17473,18 @@ msgid "&Format"
 msgstr "&Biçim"
 
 #: src/tracks/playabletrack/wavetrack/ui/WaveTrackControls.cpp
-#, fuzzy
 msgid "Changing sample format"
-msgstr "Örnekleme hızı geçersiz"
+msgstr "Örnek formatın değiştirilmesi"
```

Comment:
format kullanılmaya başlansa da biçim hala çok yaygın çevirilerde. değiştirilmesi için bir neden yok

Reply:
artı çeviri hatalı. "Örnek biçim değiştiriliyor"